libpkg/pkg.h.in

```diff
@@ -340,6 +340,12 @@ typedef enum {
 } mirror_t;
 
 typedef enum {
+	REPO_BINARY = 0,
+	REPO_DEBIAN,
+
+} type_t;
```

Comment:
I would prefer either pkg_repo_type_t or repo_type_t because type_t is too generic